testing/validation.ts

```diff
@@ -251,10 +251,16 @@ function tryParseEmail(result: unknown, schema: BaseStringSchema): ValidationErr
 function tryParseSlider(result: unknown, schema: SliderSchema | ProgressBarSchema) {
   const value = result as number;
   const {minimum, maximum} = schema;
-  if (typeof minimum !== 'string' && value < (minimum ?? 0)) {
+  if (typeof minimum === 'string') {
+    return {message: `Slider minimum value must be a number, but got string: "${minimum}".`};
```

Comment:
I don't think a string value here is technically disallowed?

My understanding is that the slider minimum value can basically be a formula which is why it has a type of `string | number`, even though nothing else in packs SDK allows people to embed formulas.

I think maybe @jonathan-codaio knows more context on sliders with string min/max values?...

Reply:
Ok I can just revert this file, it's an unrelated change.